tests/chart_tests/test_fluentd.py

```diff
@@ -10,6 +10,13 @@
     supported_k8s_versions,
 )
 class TestFluentd:
+
+    @staticmethod
+    def common_tests_daemonset(doc):
+        """Test things common to all daemonsets."""
+        assert doc["kind"] == "DaemonSet"
+        assert doc["metadata"]["name"] == "release-name-fluentd"
+
```

Comment:
Unrelated to the changes so far, the docstring on lines 21-22 needs to be updated.

Reply:
i created this function to call them verify the kind and apiVersion